src/js/flows/confirmUnload.js

```diff
@@ -3,14 +3,18 @@ import {isEmpty} from "lodash"
 
 import type {Thunk} from "../state/types"
 import Handlers from "../state/Handlers"
+import Spaces from "../state/Spaces"
+import Tab from "../state/Tab"
 import showIngestWarning from "./showIngestWarning"
 
 export default (): Thunk => (dispatch, getState) => {
-  let spaces = Handlers.getIngestSpaceNames(getState())
+  let spaceIds = Handlers.getIngestSpaceIds(getState())
```

Comment:
I was thinking this was the best way to solve this- store id and do join on state.Spaces to get name. Good to know>

Reply:
Yes, we could provide a function to make that more ergonomic though.